webpack.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+	entry: "./client/js/lounge.js",
+	resolve: {
+		extensions: ["", ".js"]
```

Comment:
Aaah, makes sense! Do we want to allow both with and without the extension? Or force one or the other format?